validation-test/compiler_crashers_2_fixed/sr9199.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -emit-ir -o %t.ll %s
+// RUN: not %target-swift-frontend -emit-ir %s
```

Comment:
FWIW This was actually an invalid test case that we used to accept (and presumably we used to crash in the even more distant past). Since we now correctly reject it, I changed the expectation here.